src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -38,12 +38,6 @@ internal partial class SymbolSearchUpdateEngine
         public const string HostId = "RoslynNuGetSearch";
         private const string MicrosoftAssemblyReferencesName = "MicrosoftAssemblyReferences";
 
-        /// <summary>
-        /// Cancellation support for the task we use to keep the local database up to date.
-        /// When VS shuts down it will dispose us.  We'll cancel the task at that point.
-        /// </summary>
-        private readonly CancellationToken _updateCancellationToken;
```

Comment:
With the exception of logging and the entry points to RPC calls on the ServiceHub side, everything should be using the same cancellation tokens they were before.